app/src/main/java/org/eclipse/kuksa/testapp/databroker/JavaDataBrokerEngine.java

```diff
@@ -33,11 +33,13 @@
 import org.eclipse.kuksa.connectivity.databroker.request.UpdateRequest;
```

Comment:
Functional Finding: 

Setting Array Values (e.g. Vehicle.Cabin.Infotainment.SmartphoneProjection.SupportedMode) also does not work, during typing you can already always see: 
```
Could not convert value: ANDROID_AUTO to ValueCase: DOUBLE
java.lang.NumberFormatException: For input string: "ANDROID_AUTO"
	at jdk.internal.math.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2054)
	at jdk.internal.math.FloatingDecimal.parseDouble(FloatingDecimal.java:110)
	at java.lang.Double.parseDouble(Double.java:660)
	at org.eclipse.kuksa.extension.DataPointExtensionKt.createDatapoint(DataPointExtension.kt:110)
	at org.eclipse.kuksa.testapp.databroker.vssnodes.view.VssNodesViewKt$VssSignalInformation$3.invoke(VssNodesView.kt:260)
	at org.eclipse.kuksa.testapp.databroker.vssnodes.view.VssNodesViewKt$VssSignalInformation$3.invoke(VssNodesView.kt:252)
...
2024-03-22 12:16:29.571 32742-32742 DataBrokerVssNodesView  pid-32742                            V  Wrong input for VssSignal:value: 
java.lang.IllegalArgumentException: VssSupportedMode copy parameters do not match: {value=ANDROID_AUTO}
	at org.eclipse.kuksa.extension.AnyCopyExtensionKt.copy(AnyCopyExtension.kt:52)
	at org.eclipse.kuksa.extension.vss.VssNodeCopyExtensionKt.copy(VssNodeCopyExtension.kt:147)
	at org.eclipse.kuksa.extension.vss.VssNodeCopyExtensionKt.copy$default(VssNodeCopyExtension.kt:142)
	at org.eclipse.kuksa.extension.vss.VssNodeCopyExtensionKt.copy(VssNodeCopyExtension.kt:131)
	at org.eclipse.kuksa.testapp.databroker.vssnodes.view.VssNodesViewKt$VssSignalInformation$3.invoke(VssNodesView.kt:261)
	at org.eclipse.kuksa.testapp.databroker.vssnodes.view.VssNodesViewKt$VssSignalInformation$3.invoke(VssNodesView.kt:252)
...
Caused by: java.lang.IllegalArgumentException: method org.eclipse.kuksa.vss.VssSupportedMode.copy argument 1 has type java.lang.String[], got java.lang.String
	at java.lang.reflect.Method.invoke(Native Method)
	at kotlin.reflect.jvm.internal.calls.CallerImpl$Method.callMethod(CallerImpl.kt:97)
	at kotlin.reflect.jvm.internal.calls.CallerImpl$Method$Instance.call(CallerImpl.kt:113)
	at kotlin.reflect.jvm.internal.KCallableImpl.callDefaultMethod$kotlin_reflection(KCallableImpl.kt:207)
	at kotlin.reflect.jvm.internal.KCallableImpl.callBy(KCallableImpl.kt:112)
	at org.eclipse.kuksa.extension.AnyCopyExtensionKt.copy(AnyCopyExtension.kt:50)
...
```

Reply:
Setting arrays<String> does work now if the input is correct e.g.: 

```
Vehicle.Cabin.Infotainment.SmartphoneProjection.SupportedMode
Input: ANDROID_AUTO, APPLE_CARPLAY
```

New ticket for supporting IntArray etc.: #109

Reply:
For VSS Nodes Mode: 

Works for single elements:
`Input: ANDROID_AUTO`

but not for CSV
`Input: ANDROID_AUTO, APPLE_CARPLAY` (with and without the gap after the comma)
as you mentioned here. 

However, as you said, let's do the fix in #109!

Reply:
Let me update the ticket so it is not forgotten.